### DISCUSSION TITLE
You unlocked new Achievements with private contributions! Show them off by including private contributions - what to do here

### DISCUSSION THREAD

**Discussion by fsj-digital** [community/community#54347] [New to GitHub]
dear friends

i do not know what this wants to say

You unlocked new Achievements with private contributions! Show them off by including private contributions in your Profile in settings.

see here:  https://github.com/fsj-digital

i do not know what that means

**Comment by neutrinomuon** [comment]
Hi @fsj-digital,

Hope you're doing well. This message is related to achievements and contributions on the GitHub platform. 

`https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile`

See the following link for the type of achievements you can get:

`https://github.com/Schweinepriester/github-profile-achievements`

I hope this helps you. Please, let me know
